linode/manager
# M3-5196: React Query for Volumes - Volumes Landing

## Description 📝

- The goal with **part 1** is to get the Volumes Landing and all associated actions such as attach, delete, etc working

## How to test 🧪

- Test **only** Volumes Landing features at `http://localhost:3000/volumes`
  - Check all actions/mutations possible in the Volumes Landing Action Menus
  - Check that Landing table updates accordingly following actions such as attach, delete, clone, etc...
